src/components/KSlideout/KSlideout.vue

```diff
@@ -55,10 +55,8 @@
         </button>
 
         <div class="content">
-          <KCard border-variant="noBorder">
-            <template #body>
-              <slot />
-            </template>
+          <KCard>
```

Comment:
**Question**: Should the border style be reset for the use in KSlideout specifically? Without it, we’ll get double borders like this:

![image](https://github.com/Kong/kongponents/assets/5774638/62140121-7128-4bc0-8b8f-fefa0d021a9c)

Reply:
Good catch, fixed